frontend/src/constants/notificationTypes.js

```diff
@@ -41,7 +41,7 @@ const NOTIFICATION_TYPES = {
   EXCLUSION_REPORT_RECOMMENDED_FOR_REJECTION_MANAGER: 'Exclusion Report Recommended for Rejection - Manager',
   EXCLUSION_REPORT_ACCEPTED: 'Exclusion Report Accepted',
   EXCLUSION_REPORT_REJECTED: 'Exclusion Report Rejected',
-  EXCLUSION_REPORT_REQUESTED_SUPPLEMENTAL: 'Exclusion Report Requested Supplemental'
+  EXCLUSION_REPORT_REQUESTED_SUPPLEMENTAL: 'Exclusion Report Supplemental Requested'
```

Comment:
Also seems awkward

Reply:
Ya, it is intentional. I just copied the suggestion from trello.